### PR TITLE
Stabilize Microsoft.NETCore.App to 1.0.3.

### PR DESCRIPTION
**Pull request by schellap**

### CODE REVIEW

**Comment by schellap** [comment]
@gkhanna79 PTAL

Cc @livarcocc @leecow

File: pkg/projects/Microsoft.NETCore.App/project.json [hunk 1]
```diff
@@ -40,8 +40,8 @@
     "System.Threading.Thread": "4.0.0",
     "System.Threading.ThreadPool": "4.0.10",
     "Libuv": "1.9.1",
-    "Microsoft.NETCore.Platforms": "1.0.2-servicing-24703-01",
```
Comment (gkhanna79) [inline pkg/projects/Microsoft.NETCore.App/project.json]: Please undo these as they will be updated by https://github.com/dotnet/core-setup/pull/616. You should rebase once that PR is merged.

**Comment by gkhanna79** [comment]
Modulo a comment, LGTM. Please try the installers to confirm the UI strings look good :)

File: pkg/projects/Microsoft.NETCore.App/project.json (hunk 1 above)
Comment (schellap) [inline pkg/projects/Microsoft.NETCore.App/project.json]: It should be able to rebase as is, I had to do this to get past the package issues.

**Comment by gkhanna79** [comment]
Please send email to shiproom about this.

**Comment by gkhanna79** [comment]
@schellap The earlier message is for you.

**Pull request merged**